lib/fs.js

```diff
@@ -1180,26 +1180,26 @@ function chmodSync(path, mode) {
 function lchown(path, uid, gid, callback) {
   callback = makeCallback(callback);
   path = getValidatedPath(path);
-  validateInteger(uid, 'uid', -1, kMaxUserId);
-  validateInteger(gid, 'gid', -1, kMaxUserId);
+  validateInteger(uid, 'uid', { min: -1, max: kMaxUserId });
```

Comment:
these changes seem wholly unrelated to QUIC, is it because validators were changed?

Reply:
Yes. The validators were updated to support some additional semantics on the QUIC side. These can be separated out.